### PR TITLE
add gnomad r3.0

### PR DESCRIPTION
**Pull request by at7**

### CODE REVIEW

**Comment by at7** [comment]
Display on my sandbox: http://ves-hx2-76.ebi.ac.uk:5040/Homo_sapiens/Variation/Population?db=core;r=1:230845294-230846294;v=rs699;vdb=variation;vf=179#gnomadgenomes_anchor

File: modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json [hunk 1]
```diff
@@ -135,52 +135,61 @@
...
     {
-      "id": "gnomADg_GRCh38",
-      "description": "Genome Aggregation Database genomes r2.1",
+      "id": "gnomADg_r3.0_GRCh38",
+      "description": "Genome Aggregation Database genomes r3.0",
       "species": "homo_sapiens",
       "assembly": "GRCh38",
       "type": "remote",
-      "filename_template": "ftp://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh38/variation_genotype/gnomad/r2.1/genomes/gnomad.genomes.r2.1.sites.grch38.chr###CHR###_noVEP.vcf.gz",
+      "filename_template": "ftp://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh38/variation_genotype/gnomad/r3.0/genomes/gnomad.genomes.r3.0.sites.chr###CHR###_trimmed_info.vcf.bgz",
```
Comment (helensch) [inline modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json]: Are these files only made available on release of 100?

File: modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json [hunk 2]
```diff
@@ -348,6 +357,34 @@
         }
       }
     },
+    { "id": "esp_GRCh37",
+      "species": "homo_sapiens",
+      "assembly": "GRCh37",
+      "type": "remote",
+      "filename_template": "ftp://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh37/variation_genotype/ESP6500SI-V2-SSA137.vcf.gz",
```
Comment (helensch) [inline modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json]: Is this file only made available at time of release?

File: modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json [hunk 3]
```diff
@@ -512,179 +549,179 @@
       "filename_template" : "ftp://ftp.ebi.ac.uk/pub/databases/eva/PRJEB22988/Svardal_et_al_2017_vervet_monkey_SNPs_imputed_phased.vcf.gz"
     },
     {
-      "id": "100_mammals.gerp_conservation_score",
+      "id": "103_mammals.gerp_conservation_score",
       "species": "bos_taurus",
       "assembly": "ARS-UCD1.2",
       "type": "remote",
-      "filename_template" : "ftp://ftp.ensembl.org/pub/release-99/compara/conservation_scores/100_mammals.gerp_conservation_score/gerp_conservation_scores.bos_taurus.ARS-UCD1.2.bw",
+      "filename_template" : "ftp://ftp.ensembl.org/pub/release-99/compara/conservation_scores/103_mammals.gerp_conservation_score/gerp_conservation_scores.bos_taurus.ARS-UCD1.2.bw",
```
Comment (helensch) [inline modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json]: Should the directory structure be release-100 for the gerp files?

File: modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json (hunk 1 above)
Comment (at7) [reply modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json]: yes

File: modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json (hunk 3 above)
Comment (at7) [reply modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json]: fixed

File: modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json (hunk 2 above)
Comment (at7) [reply modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json]: yes

File: modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json (hunk 3 above)
Comment (at7) [reply modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json]: will be fixed soon. I need to fix a merge conflict first

**Comment by at7** [comment]
merged into master and release/100

**Pull request merged**